vendor/vtd/Makefile

```diff
@@ -8,13 +8,11 @@ include ../../Makefile.package
 export:
 	# Only try to export the vtd sources if they are actually there.
 	@( \
-		cat conanfile.py \
-		| sed -rne 's/^.*"(.*\.tgz)"$$/\1/p' \
-		| while read file; do \
-			if [ ! -f $$file ]; then \
-				echo "Missing source tgz: $$file"; \
-				echo "Skipping export of: ${PACKAGE_FQN}"; \
-				return 1; \
-			fi \
-		done \
+		file=$$(cat conanfile.py \
```

Comment:
Best merge these two lines that make up the `$(...)` otherwise it's a little confusing. :-)